src/sass/patterns/_form-elements.scss

```diff
@@ -70,3 +70,17 @@
     }
   }
 }
+
+[data-form='search-issues'] {
+  gap: $spacing-x-small;
+  grid-template-rows: min-content auto;
+
+  .slds-form-element {
+    grid-column: 1;
+    grid-row: 1 / span 2;
```

Comment:
@dvdherron if we choose to rely on the LDS grid, we wouldn't need any of these styles. 

If we do want to stick with our custom grid, then 
 I'm not sure this needs to specify the grid-column since it seems each item goes in document order. Also, I think grid-row could be simplified to `grid-row: span 2`

Reply:
removed